### ISSUE TITLE
Rancher logging 3.15.0 fluentd pods go into crashloop with 2.5-head

### ISSUE THREAD

**Issue by ronhorton** [rancher/rancher#35961]
**Rancher Server Setup**
- Rancher version: `2.5-head Commit ID 777b5d4`
- Installation option (Docker install/Helm Chart): `Docker`

**Information about the Cluster**
Downstream node driver; digital ocean; 

**Describe the bug**
fluentd pods are in crashloop and there are ruby errors in the logs
```

2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/lib/fluent/engine.rb:105:in `configure'
--
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/lib/fluent/engine.rb:80:in `run_configure'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/lib/fluent/supervisor.rb:714:in `block in run_worker'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/lib/fluent/supervisor.rb:966:in `main_process'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/lib/fluent/supervisor.rb:706:in `run_worker'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/lib/fluent/command/fluentd.rb:364:in `<top (required)>'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:83:in `require'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/lib/ruby/gems/2.7.0/gems/fluentd-1.13.3/bin/fluentd:15:in `<top (required)>'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/bin/fluentd:23:in `load'
Thu, Dec 23 2021 2:31:43 pm | 2021-12-23 19:31:43 +0000 [error]: fluent/supervisor.rb:965:main_process: /usr/bin/fluentd:23:in `<main>'

```

**To Reproduce**
1. On downstream install logging v2 3.15.0 in system
2. configure output - I used elasticsearch - but i've seen this error before configuring output
3. Go to cluster explorer > Pods
4. Observe pods in cattle-logging and cattle-logging-system

**Result**
pods crash; no log output to elasticsearch

**Expected Result**
all pods running with no crashes/restarts; logs flowing to elasticsearch

**Screenshots**
<img width="1015" alt="Screen Shot 2021-12-23 at 2 27 26 PM" src="https://user-images.githubusercontent.com/89212856/147284309-9e4b6b1b-a61a-4fa2-94f1-a5a965f89882.png">

**Additional context**

**Comment by brendarearden** [comment]
@ronhorton what version of kubernetes is the downstream cluster running?

**Comment by ronhorton** [comment]
k8s version downstream: 1.20.14

**Comment by ronhorton** [comment]
I'm no longer seeing the crashing pods as of `v2.5-07a745a`

**Comment by brendarearden** [comment]
Unable to recreate - sending to test

**Comment by ronhorton** [comment]
**Issue is no longer present as of `v2.5-07a745a`**

1. On downstream install logging v2 3.15.0 in system
2. configure output - I used elasticsearch - but i've seen this error before configuring output
3. Go to cluster explorer > Pods
4. Observe pods in cattle-logging and cattle-logging-system

I repeated this test on 3 downstream clusters. 
Both versions 3.15.0 and 3.9.4 of logging.
Also verified logging v1 working as expected.
